pkg/security/secl/model/model.go

```diff
@@ -146,7 +146,7 @@ type BaseEvent struct {
 type CGroupID string
 
 // CGroupFlags represents the flags of a cgroup
-type CGroupFlags uint64
+type CGroupFlags CGroupManager
 
```

Comment:
could we move this type to model_unix.go too?

Reply:
If I remember correctly, that didn't work because CGroupFlags is also used by `pkg/security/common/containerutils/utils.go`

Reply:
Ok!